Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
Should we use one of these images https://registry.suse.com/static/suse/sle15/index.html